src/app/tests/suites/certification/Test_TC_OO_2_1.yaml

```diff
@@ -52,22 +52,70 @@ tests:
       command: "readAttribute"
       attribute: "OnTime"
       response:
+          saveAs: 'initial-On-time'
           constraints:
               type: int16u
               minValue: 0
               maxValue: 65535
 
-    - label: "Step 5: TH reads the OffWaitTime attribute from the DUT"
+    - label: "Step 5: TH writes the OnTime attribute to the DUT"
+      PICS: OO.S.A4001
+      command: "writeAttribute"
+      attribute: "OnTime"
+      arguments:
+       value: 30
+
+    - label: "Step 6: TH reads the OnTime attribute from the DUT"
+      PICS: OO.S.A4001
+      command: "readAttribute"
+      attribute: "OnTime"
+      response:
+          constraints:
+              type: int16u
+              value: 30
+
+    - label: "Step 7: TH writes the OnTime attribute to as 'initial-On-time' to restore the original value"
+      PICS: OO.S.A4001
+      command: "writeAttribute"
+      attribute: "OnTime"
+      arguments:
+       value: 'initial-On-time'
+
+    - label: "Step 8: TH reads the OffWaitTime attribute from the DUT"
       PICS: OO.S.A4002
       command: "readAttribute"
       attribute: "OffWaitTime"
       response:
+          saveAs: 'initial-Off-waittime'
           constraints:
               type: int16u
               minValue: 0
               maxValue: 65535
 
-    - label: "Step 6: TH reads the StartUpOnOff attribute from the DUT"
+    - label: "Step 9: TH writes the OffWaitTime attribute to the DUT"
+      PICS: OO.S.A4002
+      command: "writeAttribute"
+      attribute: "OffWaitTime"
+      arguments:
+       value: 30
+
+    - label: "Step 10: TH reads the OffWaitTime attribute from the DUT"
+      PICS: OO.S.A4002
+      command: "readAttribute"
+      attribute: "OffWaitTime"
+      response:
+          constraints:
+              type: int16u
+              value: 30
```

Comment:
This is causing a test failure because value has to be defined at the same level as constraints. Did you run the test locally? 

See https://project-chip.github.io/connectedhomeip-doc/testing/yaml.html#running-yaml-tests